restapi/src/main/java/io/stargate/web/docsapi/service/query/search/db/impl/PathSearchQueryBuilder.java

```diff
@@ -42,6 +44,12 @@ public PathSearchQueryBuilder(List<String> path) {
     this.path = path;
   }
 
+  private List<String> unescape(List<String> escaped) {
```

Comment:
same here, no unesaping in the query builders, paths should already be unescaped.. This is a non-needed overhead for the query builders..

Reply:
Can be resolved.